### DISCUSSION TITLE
espixelstick push button for playlist

### DISCUSSION THREAD

**Discussion by mc-sketchy** [forkineye/ESPixelStick#615] [Q&A] [answered]
is it possible to add a push button to espixelstick that would cycle through the fseq files on the sd card? or perhaps go to next playlist item on a playlist?

**Comment by MartinMueller2003** [comment, marked as answer]
Yes. Please put in your request as an 'issue' and describe what you are trying to accomplish and then describe how you would like it to work (those are two separate descriptions). We had some push button functionality in the V4 code but there was not a lot of interest in using it.

**Reply by mc-sketchy** [reply]
Excellent. Will do.  Thank you Martin

**Reply by hell1337hound** [reply]
i'm also keen for this :)
